client/js/components/Query.js

```diff
@@ -55,11 +55,11 @@ export default class Query extends React.Component {
     var self = this;
     self.setState({ showPlot: true });
     if (self.state.stats.length === 0) {
-      Request.get("/api/stats")
+      Request.get("/api/wage_list")
              .query(self.state.filters)
              .end(function (err, res) {
                if (res.body.status === "Okay") {
-                 self.setState({ stats: res.body.data.hist });
+                 self.setState({ stats: res.body.data.wages });
```

Comment:
Let's change the name from `stats` here to something more descriptive of what is being stored.